pages/common/protoc.md

```diff
@@ -0,0 +1,19 @@
+# protoc
+
+> Parse Google Protobuf `.proto` files and generate output in the specified language.
+
+- Generate Python code from a `.proto` file:
+
+`protoc --python_out={{output_directory}} {{input_file}}`
+
+- Generate Java code from a `.proto` file that imports other `.proto` files:
+
+`protoc --java_out={{output_directory}} --proto_path={{import_search_path}} {{input_file}}`
+
+- Generate Java code from a `.proto` file that imports other `.proto` files:
+
+`protoc --java_out={{output_directory}} --proto_path={{import_search_path}} {{input_file}}`
+
+- Generate code for multiple languages:
+
+`protoc --csharp_out={{c#_output_diry}} --js_out={{javascript_output_dir}} {{input_file}}`
```

Comment:
Let's use a consistent naming convention. Change to `path/to/c#_output_directory` and `path/to/js_output_directory`.